common/tools/dev-tool/src/commands/run/extract-api.ts

```diff
@@ -123,7 +112,16 @@ function extractApi(
   }
 }
 
-export default leafCommand(commandInfo, async (options) => {
+interface ApiJson {
+  metadata: Record<string, unknown>;
+  members: { kind: string; name: string }[];
+}
+
+async function loadApiJson(fullPath: string): Promise<ApiJson> {
+  return JSON.parse((await readFile(fullPath)).toString("utf-8")) as ApiJson;
+}
+
+export default leafCommand(commandInfo, async (_options) => {
```

Comment:
Do you have to use mark options if you're not using it?

Reply:
Right, can be removed

Reply:
Fixed.